accumulo-store/src/test/java/gaffer/accumulostore/utils/AccumuloPropertyNames.java

```diff
@@ -21,4 +21,5 @@
 public class AccumuloPropertyNames extends TestPropertyNames {
     public static final String COLUMN_QUALIFIER = "columnQualifier";
     public static final String TIMESTAMP = "timestamp";
+    public static final String TEST_OPTION_KEY = "testOption";
```

Comment:
This test option key isn't a property name, it looks like it is just used as an operation option key, so it shouldn't really be in the AccumuloPropertyNames constants class.